README.md

```diff
@@ -68,15 +47,11 @@ If you want to iterate on *just* the helper go code and there tests, do one roun
 
 Then just iterate using `task apis:generate` and `task apis:test`
 
-#### For avid
```

Comment:
I assume your not saying "we don't support this for avid" rather you are saying it works for *any* consuming package ? Do I have that right ?

Reply:
yes - the deleted text was written when we only had avid
its any repo now

Reply:
will do